### ISSUE TITLE
CosyVoice2的流式推理，模型加载lod_trt=True的情况下，对推理应该不会有提升吧？

### ISSUE THREAD

**Issue by russell-shu** [FunAudioLLM/CosyVoice#1020]
p = threading.Thread(target=self.llm_job,
                             args=(text, prompt_text, llm_prompt_speech_token, llm_embedding, this_uuid))
        p.start()
        if stream is True:
            token_offset = 0
            while True:
                time.sleep(0.1)

                if len(self.tts_speech_token_dict[this_uuid]) - token_offset >= self.token_hop_len + self.flow.pre_lookahead_len:
                    current_token_count = len(self.tts_speech_token_dict[this_uuid])
                    process_start = time.time()
                    this_tts_speech_token = torch.tensor(self.tts_speech_token_dict[this_uuid][
                                                         :token_offset + self.token_hop_len + self.flow.pre_lookahead_len]).unsqueeze(
                        dim=0)
                    this_tts_speech = self.token2wav(token=this_tts_speech_token,
                                                     prompt_token=flow_prompt_speech_token,
                                                     prompt_feat=prompt_speech_feat,
                                                     embedding=flow_embedding,
                                                     uuid=this_uuid,
                                                     token_offset=token_offset,
                                                     finalize=False)
                    token_offset += self.token_hop_len

                    process_time = time.time() - process_start
                    # 计算本周期内LLM的新增token
                    new_tokens =  len(self.tts_speech_token_dict[this_uuid]) - current_token_count

             
                    logging.info(
                        f"处理耗时{process_time:.3f}s | "
                        f"LLM在本次音频推理中新增{new_tokens}个token | "
                    )
                    yield {'tts_speech': this_tts_speech.cpu()}
                 
                    token_offset += self.token_hop_len

                if self.llm_end_dict[this_uuid] is True and len(self.tts_speech_token_dict[
                                                                    this_uuid]) - token_offset < self.token_hop_len + self.flow.pre_lookahead_len:

                    break
            p.join()

在满足len(self.tts_speech_token_dict[this_uuid]) - token_offset >= self.token_hop_len + self.flow.pre_lookahead_len的一次循环中，llm_job新增的token量是小于self.token_hop_len + self.flow.pre_lookahead_len，主循环线程是一直在等待线程p返回token值(论文中也提到)。那么代码里的flow模型加速还有必要吗？

打印如下：
2025-02-26 17:03:23,281 INFO [Processing] 单次数据处理耗时: 0.150s
2025-02-26 17:03:23,281 INFO 处理耗时0.150s | LLM在本次音频推理中新增8个token | 
2025-02-26 17:03:23,281 INFO yield speech len 1.84, rtf 0.517475734586301
chunk[tts_speech]: torch.Size([1, 44160])
2025-02-26 17:03:24,537 INFO [Processing] 单次数据处理耗时: 0.152s
2025-02-26 17:03:24,537 INFO 处理耗时0.152s | LLM在本次音频推理中新增9个token | 
2025-02-26 17:03:24,537 INFO yield speech len 2.0, rtf 0.6268883943557739
chunk[tts_speech]: torch.Size([1, 48000])
2025-02-26 17:03:25,796 INFO [Processing] 单次数据处理耗时: 0.154s
2025-02-26 17:03:25,796 INFO 处理耗时0.154s | LLM在本次音频推理中新增9个token | 
2025-02-26 17:03:25,796 INFO yield speech len 2.0, rtf 0.627973198890686
chunk[tts_speech]: torch.Size([1, 48000])
2025-02-26 17:03:26,963 INFO [Processing] 单次数据处理耗时: 0.164s
2025-02-26 17:03:26,963 INFO 处理耗时0.164s | LLM在本次音频推理中新增10个token | 
2025-02-26 17:03:26,963 INFO yield speech len 2.0, rtf 0.5826324224472046
chunk[tts_speech]: torch.Size([1, 48000])
2025-02-26 17:03:27,818 INFO yield speech len 0.68, rtf 1.2529215391944435
